### PR TITLE
setting minimum_height to height in clipboard.py

### PR DESCRIPTION
**Pull request by bhaveshAn**
examples/miscellaneous/clipboard.py started working after this correction

### CODE REVIEW

**Comment by KeyWeeUsr** [comment]
In 1.9.1+ this example works correctly. Any particular example that doesn't work would be nice.

Btw, what OS and Kivy version do you have? What about providers?

**Comment by bhaveshAn** [comment]
Working on OS Windows 10 and kivy 1.9.1
Clipboard provider winctypes

**Comment by KeyWeeUsr** [comment]
Apparently there was no `minimum_height` in `BoxLayout` yet. I wonder how it worked before...

Sorry for misunderstanding and thanks for noticing! 👍

**Comment by bhaveshAn** [comment]
Np @KeyWeeUsr

File: examples/miscellaneous/clipboard.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
     orientation: 'vertical'
     BoxLayout:
         size_hint_y: None
-        height: self.minimum_height
+        height: self.height
```
Comment (dessant) [inline examples/miscellaneous/clipboard.py]: The property references itself and the value will just stay `100`. If compatibility with older versions is needed, GridLayout could be used.

**Pull request merged**